experimental/manifests/base/argo/workflow-controller-configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+data:
+  config: |
+    {
+    executorImage: argoproj/argoexec:v2.2.0,
+    artifactRepository:
+    {
```

Comment:
Is this indent correct and do we need curly braces?

Reply:
it's just to align with the existing one.